FogCreek/Glitch-Community
# 🌪🌈 Complete Farmhouse: replace Arrow with Icon

./src/components/collection/collection-item.js:9/73
./src/components/collection/collection-item-small.js:9/73
./src/components/collection/container.js:9/73
./src/components/collections-list/more-collections.js:9/114
./src/components/containers/results-list.js:9/79
./src/components/new-stuff/new-stuff-article.js:9/125
./src/components/questions/index.js:9/125
./src/components/recent-projects/index.js:9/73
./src/components/related-projects/index.js:9/168
./src/components/search-form/autocomplete.js:9/126
./src/presenters/pages/home-v2/banner.js:9/73
./src/presenters/pages/home-v2/index.js:9/178

## Links
* https://complete-farmhouse.glitch.me/
* https://app.zenhub.com/workspaces/community-5d07d1359dee6059a9688176/issues/fogcreek/glitch-community-work/392

## Changes:
* the Arrow component is really just a (not so) special case of Icon, so just replace it everywhere

## How To Test:
* Click around, look for arrows (there are a bunch on the logged-out homepage, also handraise has one)
* Check spacing, how they look on different browsers

## Things left to do before deploying:
- Marvel at how `complete-farmhouse` had the checkin of tornado + rainbow? are we off to see the wizard?
